packages/core/src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -254,6 +254,26 @@ export default class BraintreeHostedForm {
         }
     }
 
+    private _mapErrors(fields: any): any {
+        const errors = {};
+
+        if (fields) {
+            for (const [key, value] of Object.entries(fields)) {
+                const { isValid, isEmpty, isPotentialyValid } = value as any;
```

Comment:
If you make all [this](https://github.com/bigcommerce/checkout-sdk-js/pull/1883/files#r1121657345) changes value any type can be updated with BraintreeFormErrorData

```suggestion
                const { isValid, isEmpty, isPotentialyValid } = value as BraintreeFormErrorData;
```

Reply:
Also will try to avoid using `as` keyword and use a type guard, for reference https://blog.logrocket.com/how-to-use-type-guards-typescript